subiquity/models/subiquity.py

```diff
@@ -41,12 +42,24 @@ def setup_yaml():
 setup_yaml()
 
 
+def get_all_groups(dry_run):
+    command = ['chroot', '/target', 'getent', 'group']
```

Comment:
Shouldn't /target be part of one of the models, maybe the base subiquity.model ?

Reply:
Yes, done, and cleaned up all the other hardcoded paths while I was at it.